src/sql/parts/notebook/notebook.component.ts

```
@@ -173,6 +175,14 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	private disconnect() {
+		if (this._model.defaultKernel.display_name === notebookConstants.SQL) {
+			if (this._model.activeCell && this._model.activeCell.cellType === CellTypes.Code && this._model.activeCell.cellUri) {
+				this.connectionManagementService.disconnect(this._model.activeCell.cellUri.toString());
```

Aren't these awaitable? You have to catch errors, even if swallow.